pkg/controllers/vdb/init_db.go

```diff
@@ -65,6 +70,11 @@ type GenericDatabaseInitializer struct {
 	ConfigurationParams *vtypes.CiMap
 }
 
+type GcpSecret struct {
```

Comment:
Lets not export this struct. Also, secret is too generic. Lets call out the fact that it's for credentials.
```suggestion
type gcpCredentialSecret struct {
```

Reply:
This comment is obsolete now as I changed it to map[string]string{} and this struct is no more present